app/src/sharedTest/java/org/oppia/app/settings/profile/ProfileResetPinActivityTest.kt

```diff
@@ -165,6 +168,17 @@ class ProfileResetPinActivityTest {
       onView(allOf(withId(R.id.error_text), isDescendantOfA(withId(R.id.input_confirm_pin)))).check(matches(withText("")))
     }
   }
+  @Test
+  fun testProfileResetPinActivity_startActivitywithAdmin_configurationLandscape_isCorrect() {
+    ActivityScenario.launch<ProfileResetPinActivity>(ProfileResetPinActivity.createProfileResetPinActivity(context, 0, true)).use{
+      onView(ViewMatchers.isRoot()).perform(orientationLandscape())
```

Comment:
Refractor it as below to make better readability:

`onView(isRoot()).perform(orientationLandscape())`

and subsequently imports will change and thus delete unused imports